src/tools/tool-options.h

```diff
@@ -33,88 +34,98 @@ struct ToolOptions : public Options {
+               explicitFeatures = true;
              })
         .add("--enable-mutable-globals", "", "Enable mutable globals",
              Options::Arguments::Zero,
              [this](Options *o, const std::string& arguments) {
                passOptions.features.setMutableGlobals();
+               explicitFeatures = true;
              })
         .add("--disable-mutable-globals", "", "Disable mutable globals",
              Options::Arguments::Zero,
              [this](Options *o, const std::string& arguments) {
                passOptions.features.setMutableGlobals(false);
+               explicitFeatures = true;
              })
         .add("--enable-nontrapping-float-to-int", "",
              "Enable nontrapping float-to-int operations",
              Options::Arguments::Zero,
              [this](Options *o, const std::string& arguments) {
                passOptions.features.setTruncSat();
+               explicitFeatures = true;
              })
         .add("--disable-nontrapping-float-to-int", "",
              "Disable nontrapping float-to-int operations",
              Options::Arguments::Zero,
              [this](Options *o, const std::string& arguments) {
                passOptions.features.setTruncSat(false);
+               explicitFeatures = true;
              })
         .add("--enable-simd", "",
              "Enable SIMD operations and types",
              Options::Arguments::Zero,
              [this](Options *o, const std::string& arguments) {
                passOptions.features.setSIMD();
+               explicitFeatures = true;
              })
         .add("--disable-simd", "",
              "Disable SIMD operations and types",
              Options::Arguments::Zero,
              [this](Options *o, const std::string& arguments) {
                passOptions.features.setSIMD(false);
+               explicitFeatures = true;
              })
         .add("--enable-bulk-memory", "",
              "Enable bulk memory operations",
              Options::Arguments::Zero,
              [this](Options *o, const std::string& arguments) {
                passOptions.features.setBulkMemory();
+               explicitFeatures = true;
              })
         .add("--disable-bulk-memory", "",
              "Disable bulk memory operations",
              Options::Arguments::Zero,
              [this](Options *o, const std::string& arguments) {
                passOptions.features.setBulkMemory(false);
+               explicitFeatures = true;
              })
-        .add("--no-validation", "-n", "Disables validation, assumes inputs are correct",
+        .add("--no-validation", "-n",
+             "Disables validation, assumes inputs are correct",
              Options::Arguments::Zero,
              [this](Options* o, const std::string& argument) {
                passOptions.validate = false;
              });
-        ;
-  }
-
-  FeatureSet getFeatures() const {
```

Comment:
why was this removed?

Reply:
It didn't seem necessary since most other properties of the passOptions and related structs are accessed directly. I could alternatively have added a corresponding `setFeatures` method.

Reply:
Another option is to return the features by reference here, perhaps.